.github/pull_request_template.md

```diff
@@ -4,6 +4,15 @@
   - Note:
     - Consider running `automation/all_tests.sh` to execute these tests locally.
```

Comment:
The old script ran basically all of the `mode` values except `changes`.  It would be pretty easy to add a an `all-tests` mode that did something very similar.

Reply:
Okay, I'm going to remove this line. The only reason it's here is to short-circuit the CI feedback loop. If it's more convenient to wait on CI then there's no added value here.